Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php

```diff
@@ -490,6 +490,7 @@ public function getDefaultColumns()
             'customer.groupKey as customergroup',
             'customer.newsletter as newsletter',
             'customer.affiliate as affiliate',
+            'billing.number as customerNumber',
```

Comment:
and now you could use `customer.number`, couldn't you?

Reply:
You are right again. I've updated the pr.